python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_relative.py

```diff
@@ -642,7 +643,7 @@ def _analyse_noSOMD(
             units="kcal/mol",
             software=engine,
             dir=work_dir,
-            prefix=prefix,
```

Comment:
So is that the main change in behaviour between 1 and 2?

Reply:
Yes, the dir accepts a real directory so 'lambda_*' is not accepted as dir input and has to go to prefex